dmscripts/notify_buyers_to_award_closed_briefs.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, date, timedelta
+
+import dmapiclient
+from dmutils.email.dm_notify import DMNotifyClient
+from dmutils.email.exceptions import EmailError
```

Comment:
`DMNotifyClient` and `EmailError` are both imported by the `__init__.py` of `dmutils.email` so you can just do `from dmutils.email import DMNotifyClient, EmailError`

Reply:
Good to know! Perhaps I should change in (wherever the place I copied the import from was) too...

Reply:
Aha - scripts is still using utils v.27.1.2. I don't think there's any harm bumping it up to v28?

Reply:
Nope, wouldn't have thought so. The breaking change is about decoding invitation tokens.